config/requests.yml.tmpl

```diff
@@ -6,6 +6,8 @@ defaults: &defaults
   proxy_base: http://library.princeton.edu/resolve/lookup?url=
   stackmap_base: https://library.princeton.edu/utils/map
   voyager_api_base: http://catalog.princeton.edu:7014
+  voyager_ub_id: 1@DB1
+  pulsearch_base: https://pulsearch.princeton.edu
```

Comment:
This config value is their for the benefit of the requests engine. If the engine was mounted in an application other than OL and wanted to query the OL API it would need to know the base URL for OL.